StudCity.Frontend/src/components/Scroller/index.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 
 const Scroller = ({ children }) => {
 	return (
-		<div className='h-full w-full overflow-y-auto'>
+		<div className='h-full w-full flex overflow-y-auto'>
```

Comment:
Are you sure that this does not affect other components? You might want to add a className property to this component